README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_gallery_DOM/)
+    - [DEMO LINK](https://katerynashylina.github.io/js_gallery_DOM/)
```

Comment:
Don't forget to add this link in PR description

Reply:
oopsy, totally forgot